Manhattan-Pizza-and-Pub.md

```diff
@@ -0,0 +1,19 @@
+# Manhattan Pizza & Pub
+
+##Address:## 167 Main St, Burlington, VT 05401
```

Comment:
this syntax doesn't follow the style guide -- please use asterisks instead of hash marks. See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet for more info about Markdown